jupyterlab_polus_render/README.md

```diff
@@ -1,119 +1,152 @@
-|----------------|---------------|---------------|----------------|-----------|-----|
-| Static | :heavy_check_mark:  | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark:
+# Adding a static build of Polus Render
+- Remove all existing files in `~/render-server-ext/static/render-ui/`. 
+- Run `npx nx build render-ui` in the root of your Polus Render folder
+- Transfer generated files from `~/Polus Render/dist/apps/render-ui/` into `~/render-server-ext/static/render-ui/`. 
 
-# Drag & Drop Demo
-<img src="images/drag-drop.gif"/>
+# API Endpoints
+- `/render/default/(.*)`: Help on usage of extension
+- `/static/render/render-ui/index.html`: Serves static build of Polus Render.
+- `/render/file/(.+)`: Serves files at a specfied path. Does not serve directories.
 
-# Sample usage
-``` Python
-from polus.polus_render import render
+# Examples
+For the following examples, JupyterLab is ran locally at `https://localhost:7832/lab`
 
-# pathlib and urllib are built-ins
-from urllib.parse import urlparse
-from pathlib import Path
 
-# Make sure to keep track of your JupyterLab url and file root if your root is not at "/home/joyvan".
-JL_URL = urlparse("https://<JUPYTERHUB_URL>/user/<USERNAME>/user-namespaces/lab?")
+## Render-UI 
+URL: `http://localhost:7832/static/render/render-ui/index.html?imageUrl=http://localhost:7832/render/file//home/jovyan/work/images/4by4large_intensity_image.ome.tif`
 
-# Embeds an IFrame of a static build of Polus Render into Jupyter Lab, this is sufficient if your file root is "/home/joyvan/"
-render(nbhub_url=JL_URL)
+<img src="images/renderui-1.png"/>
 
-# Same as above; however, if your file root is "/Users/jeff.chen/", your invocation will require nb_root argument
-render(nbhub_url=JL_URL, \
-    nb_root=Path("/Users/jeff.chen/"))
 
-# Embeds an IFrame of a static build of Polus Render with an image hosted at "/home/joyvan/zarr files/pyramid.zarr"
-render(nbhub_url=JL_URL, \
-    image_location=Path(r"zarr files/pyramid.zarr"))
+URL: `http://localhost:7832/static/render/render-ui/index.html?imageUrl=http://localhost:7832/render/file//home/jovyan/work/images/x00_y01_p02_c1.ome.tif&overlayUrl=http://localhost:7832/render/file//home/jovyan/work/overlays/combined.json`
 
-# Embeds an IFrame of a static build of Polus Render with an image and overlay file
-render(nbhub_url=JL_URL, \
-    image_location=Path("zarr files/pyramid.zarr"), \
-    microjson_overlay_location=Path("overlay files/x00_y01_c1_segmentations.json"))
+<img src="images/renderui-2.png"/>
 
-# Embeds an IFrame of a static build of Polus Render with remote image
-render(nbhub_url=JL_URL, \
-    image_location=urlparse("https://viv-demo.storage.googleapis.com/LuCa-7color_3x3component_data.ome.tif"))
+
+## Image
+URL: `http://localhost:7832/render/file/home/jovyan/work/samples/butterfly.jpeg`
+
+<img src="images/image.png"/>
+
+```
```

Comment:
This section can be deleted along with the images, but please create a new screenshot of the extension in use and put it at the top of this README just above the Requirements section